oraclizeAPI_0.5.sol

```diff
@@ -77,760 +73,881 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+            let buflen := mload(bufptr) // Length of existing buffer data
+            let dest := add(add(bufptr, buflen), 32) // Address = buffer address + buffer length + sizeof(buffer length)
+            mstore8(dest, _data)
+            mstore(bufptr, add(buflen, 1)) // Update buffer length
         }
     }
-
     /**
-     * @dev Appends a byte to the end of the buffer. Resizes if doing so would
-     * exceed the capacity of the buffer.
-     * @param buf The buffer to append to.
-     * @param data The data to append.
-     * @return The original buffer.
-     */
-    function appendInt(buffer memory buf, uint data, uint len) internal pure returns(buffer memory) {
-        if(len + buf.buf.length > buf.capacity) {
-            resize(buf, max(buf.capacity, len) * 2);
-        }
-
-        uint mask = 256 ** len - 1;
+      *
+      * @dev Appends a byte to the end of the buffer. Resizes if doing so would
+      * exceed the capacity of the buffer.
+      * @param _buf The buffer to append to.
+      * @param _data The data to append.
+      * @return The original buffer.
+      *
+      */
+    function appendInt(buffer memory _buf, uint _data, uint _len) internal pure returns (buffer memory) {
+        if (_len + _buf.buf.length > _buf.capacity) {
+            resize(_buf, max(_buf.capacity, _len) * 2);
+        }
+        uint mask = 256 ** _len - 1;
         assembly {
-            // Memory address of the buffer data
-            let bufptr := mload(buf)
-            // Length of existing buffer data
-            let buflen := mload(bufptr)
-            // Address = buffer address + buffer length + sizeof(buffer length) + len
-            let dest := add(add(bufptr, buflen), len)
-            mstore(dest, or(and(mload(dest), not(mask)), data))
-            // Update buffer length
-            mstore(bufptr, add(buflen, len))
-        }
-        return buf;
+            let bufptr := mload(_buf) // Memory address of the buffer data
+            let buflen := mload(bufptr) // Length of existing buffer data
+            let dest := add(add(bufptr, buflen), _len) // Address = buffer address + buffer length + sizeof(buffer length) + len
+            mstore(dest, or(and(mload(dest), not(mask)), _data))
+            mstore(bufptr, add(buflen, _len)) // Update buffer length
+        }
+        return _buf;
     }
 }
 
 library CBOR {
+
     using Buffer for Buffer.buffer;
 
     uint8 private constant MAJOR_TYPE_INT = 0;
-    uint8 private constant MAJOR_TYPE_NEGATIVE_INT = 1;
+    uint8 private constant MAJOR_TYPE_MAP = 5;
     uint8 private constant MAJOR_TYPE_BYTES = 2;
-    uint8 private constant MAJOR_TYPE_STRING = 3;
     uint8 private constant MAJOR_TYPE_ARRAY = 4;
-    uint8 private constant MAJOR_TYPE_MAP = 5;
+    uint8 private constant MAJOR_TYPE_STRING = 3;
+    uint8 private constant MAJOR_TYPE_NEGATIVE_INT = 1;
     uint8 private constant MAJOR_TYPE_CONTENT_FREE = 7;

```

Comment:
Habitual, and ofc unnecessary (unless you also have a thing for code aesthetics...) I zone out & put things in length order whilst thinking... :thinking:

Reply:
I tend to find it harder to track things via this ordering, I don't know if others wish to provide input regarding it as well (although very few would be referring to this part of the code likely, and since it is copied from outside, maybe it's easier for cross-reference to keep the original order).